app/components/qualification_grade_component.rb

```diff
@@ -8,7 +8,7 @@ def grade
     if @qualification.predicted_grade?
       "#{@qualification.grade} (predicted)"
     else
-      @qualification.grade
+      @qualification.grade || 'Not entered'
```

Comment:
```suggestion
      @qualification.grade.presence || 'Not entered'
```
because there will be some existing qualifications that have an empty string value?

Reply:
Was not aware of `#presence` - good to know 👍

Reply:
Good thinking Steve, thanks for raising that.